parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -159,6 +158,11 @@ ensureContainerd {{/* containerd should not be configured until cni has been con
 ensureDocker
 {{- end}}
 
+if [ -f $VHD_LOGS_FILEPATH ]; then
+    echo "removing unused pre-pulled images"
+    cleanUpContainerImages
```

Comment:
remove L79

Reply:
should we be worried about prolonged cse execution due to the delay or disk space issue (in extreme case when vhd is almost full and we cannot install anything before cleaning up the images)? I have two possible options
1) move ensureDocker/containerd up to as the first step, but I notice containerd has dependencies on CNI, not sure if there are other dependencies that should be moved along too
2) keep the current cleanup image and redo it after ensureDocker/containerd?